src/config/dynamic-theme-fixes.config

```diff
@@ -11868,6 +11868,16 @@ CSS
 
 ================================
 
+ups.com
+
+CSS
+#ups-alertsWrap,
+#ups-skipNav {
+    background: #740 !important;
```

Comment:
Hmm hardcoded values 👀

Reply:
I use hard coded colors locally because they are fast work well for me, but I agree 100% that they may not be ideal for the community.

I use CSS overlays locally because I can create the fixes faster using my tools than using the Dark Reader UI.  I wanted to share with our community, so I converted several of them to Dark Reader syntax/format.

If you let me know a better way to handle this case, I'll make edits here and also use whatever knowledge you teach me for all future submissions.

Reply:
So for this kind of values you simply can wrap it into a `${#740}` and then check the elements again and copy the background-color value. Wrap that again into a `${pasted_value}` and use that instead. It will on your settings default to `#740` and for other users they will be slightly different to match with the user's settings.

Reply:
Thanks.  I think I'm following what you are saying.  Here's what I think you are saying... please correct any errors on my part (I corrected what I think may be one small error on your part).

1. Instead of using my custom CSS overlay to verify the code is working, use Dark Reader's Dev Tools.
2. In Dark Reader's Dev Tools, assign a value of `${#740}` to the `background` property of the element.
3. Go to Firefox's Dev Tools, and see what value gets assigned to that element.
4. Copy that value to the clipboard; let's call it `${copied_value}`
5. In Dark Reader's Dev Tools, replace `${#740}` with `$(copied_value}`.
6. In Dark Reader's Dev Tools, hit the `Apply` button.
7. If things don't looks good, adjust the color (or use a different Dark Reader supported technique).
8. Once everything looks good, copy the entire section to GitHub and create a pull request.

Did I get it right?

It's a fair amount of time/energy just to fix one site, but I'll continue to do my best to create pull requests for each site I think may have enough general interest to be helpful to others.

Reply:
> Did I get it right?

Yeah you did.

It's a very special case. As most of the time what I do is get the originial value of such property(not the one dark reader is applying currently) and throw that into `${}` and 9/10 it's perfection or I use a color that looks like it. e.g. in this cause `${orange}`, however in such case it's a bit longer.

Reply:
Thanks.  Glad I got it right. :)   I'll copy our dialogue to a text file for reference for future contributions.

I actually had a hunch that hardcoded values were a no-no, but I thought I saw quite a few items in the file that use them, so I thought maybe it was okay.  It might be worthwhile scanning through it for hardcoded values?

BTW, are there exceptions for `#FFF` and `#000` (and maybe `#777`)?  IIRC, there were multiple existing fixes that used those values, and if I'm not mistaken, you recently accepted another one of my pull requests that used one of those values.

Reply:
> so I thought maybe it was okay. It might be worthwhile scanning through it for hardcoded values?

Or they an exception which on a occasion happens when something got broken like a feature etc(qr code border). Or they where merged into the config 1+ year ago. 

> BTW, are there exceptions for #FFF and #000 (and maybe #777)? IIRC, there were multiple existing fixes that used those values, and if I'm not mistaken, you recently accepted another one of my pull requests that used one of those values.

With #FFF #000 you can just use var(--darkreader-neutral-background) and var(--darkreader-neutral-color) as they are more natural overall. 

Hmm that must be by mistake, lately got more random "merged" when I submit feedback.

Reply:
Thanks for the additional details.  Noted and appreciated.